config/config.go

```diff
@@ -73,3 +82,43 @@ func getFrontendURL() (string, error) {
 
 	return e, nil
 }
+
+// getSchedulerCfg read from kube scheduler config file path environment variable
+// named KUBE_SCHEDULER_CONFIG_PATH and convert it into *v1beta2config.KubeSchedulerConfiguration.
+func getSchedulerCfg() (*v1beta2config.KubeSchedulerConfiguration, error) {
+	e := os.Getenv("KUBE_SCHEDULER_CONFIG_PATH")
+	if e == "" {
+		return nil, nil
```

Comment:
I think it's better to return the default configuration here if KUBE_SCHEDULER_CONFIG_PATH is not set.

If we do so, we can remove this kinda logic.
- https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/80/files#diff-8f41374cc7da8c5d92c41034deebe762cc34593a7496a0000646715ab75cbd16R56-R62
- https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/80/files#diff-4741d40c952e5146eb01d0005e03250ada8a9d57c375f9f7c7026ab53cb1f7e1R50-R56

Reply:
ah, I missed it! Thank you, I'll fix it.